app/models/user.rb

```diff
@@ -38,6 +38,10 @@ def current_friend_requests
       requests[friend] = User.find(friend.from)
     end
     requests
+     #x=  FriendRequest.where(status: 0).where(user_id: current_user.id)
```

Comment:
Is this a comment for future iterations?

Reply:
No, that was from a comment on the dry run from earlier about possibily needing friend name from friend request. I'll remove and update.